po/git-novice.ja.po

```diff
@@ -7638,31 +7633,25 @@ msgid ""
 "    which  makes it much easier for other scientists\n"
 "    to use her work as starting point for their own research."
 msgstr ""
-"    like [figshare](https://figshare.com/) or\n"
-"    [Zenodo](https://zenodo.org), possibly as soon as it's collected,\n"
-"    and given its own\n"
-"    [Digital Object Identifier](https://en.wikipedia.org/wiki/"
-"Digital_object_identifier) (DOI).\n"
-"    Or the data was already published and is stored in\n"
-"    [Dryad](https://datadryad.org/).\n"
-"*   The scientist creates a new repository on GitHub to hold her work.\n"
-"*   As she does her analysis,\n"
-"    she pushes changes to her scripts\n"
-"    (and possibly some output files)\n"
-"    to that repository.\n"
-"    She also uses the repository for her paper;\n"
-"    that repository is then the hub for collaboration with her colleagues.\n"
-"*   When she's happy with the state of her paper,\n"
-"    she posts a version to [arXiv](https://arxiv.org/)\n"
-"    or some other preprint server\n"
-"    to invite feedback from peers.\n"
-"*   Based on that feedback,\n"
-"    she may post several revisions\n"
-"    before finally submitting her paper to a journal.\n"
-"*   The published paper includes links to her preprint\n"
-"    and to her code and data repositories,\n"
-"    which  makes it much easier for other scientists\n"
-"    to use her work as starting point for their own research."
+"    [figshare](https://figshare.com/)や\n"
+"    [Zenodo](https://zenodo.org)\n"
+"    などのオープンアクセスリポジトリに保存され、独自の\n"
+"    [デジタルオブジェクト識別子](https://en.wikipedia.org/wiki/"
+"Digital_object_identifier) (DOI) が与えられます。\n"
+"    または、データはすでに公開されており、\n"
+"    [Dryad](https://datadryad.org/) に保存されています。\n"
+"*   科学者は、自分の研究を保持するためにGitHubに新しいリポジトリを作成します。\n"
+"*   科学者は分析を行うときに、スクリプト (場合によってはいくつかの出力ファイルも) への変更をそのリポジトリにプッシュします。\n"
+"    科学者はまた、自分の論文用のリポジトリを使用します;\n"
```

Comment:
She doesn't create another repository for the paper, but uses the same one for the paper (unfortunately, it's not clear what "use the repository for her paper" means)
```suggestion
"    科学者はまた、自分の論文用にそのリポジトリを使用します;\n"
```